### ISSUE TITLE
Qiang Cai - Marketing Secretary

### ISSUE THREAD

**Issue by sanamorii** [HackSocNotts/nominations#40]
# Name: Qiang Cai

## Marketing Secretary

# Manifesto

![fumo](https://github.com/HackSocNotts/nominations/assets/86570135/fe9b2385-9cb0-4440-b532-4a4b1f171750)
![funny](https://hacksocnotts.co.uk/img/committee/qiang.png)
funny pictures 

Hello, my name is Qiang, a second year/incoming third year Computer Science student. From my time at university, I have really enjoyed particpating in HackNotts and the event's HackSoc hosts. I have learnt alot from them in terms of technical skill and domain knowledge that I otherwise would've never learnt through regular lectures or self-study.  

I've always wanted to contribute back and be a more integral part of HackSoc because a. I just thought it would be cool. b. I could learn more from helping to runn the society itself. c. I wanted to add the thing into the society that I thought could be a great addition. These were my main motivations for becoming Trips and Hackathons Secretary and still continues to be now that I'm running for Marketing Secretary (that and me going to hackathons lol). Now having first hand experience being apart of HackSoc's Committee as the Trips and Hackathon's Secretary and helping to organise HackNotts has given me some ideas on the things that I want to improve and strengthen in HackSoc if elected.

## A few things about me:
1. I have at least 20 shelved projects; all of which will continued to collect dust until the forseeable future.
2. I have an unhealthy obession for these funny plushes known as fumos.
3. Functional programming is both my bane and beloved.
4. I have successfully copied most of Lan Zhou Noodle's dishes.
5. The maid dress is forever encased in my wardrobe and it stares at me in disappointment everytime I open it.
6. I was the one who designed the condom graphics for HackSoc to hand out during Welcome Fair. ~~not like the members would need them but whatever :3~~

## My aims as Marketing Secretary
- **Review and update branding**; The general consensus from the committee when I brought it up, was that HackSoc's branding and logo is already good enough so it's not really warranted for a massive rebranding, however, it isn't really accessible nor flexible in use for advertising or event. I wish to update the branding of HackSoc to fix these problems but also keeps the spirit/vibe that previous committee have cultivated. 
  -  In addition; I also want to create a brand/styling guideline so that future graphics and advertisements can remain somewhat consistent and accessibile. This extends to HackNotts. 
  - sorry daudi, i will definitely get on this i was just plagued with work this semester :(
- **Better and consisent communication**; One thing I noticed is that communication for events and updates around the society is somewhat sparse and inconsistent; of course this results in lower turnout for society events and workshops which I've seen alot with Programming Cafe or the Cafe Crawls in where they just end out becoming Committee hang-outs or meetings (which I don't mind lol). Obviously there other factors that result in low turnout (i.e. coursework, exams, etc.), but I do think that if we were to focus on eariler and better marketing, that event turnout will be alot higher. Overall, I am to achieve better marketing through:
  - 1. Earlier advertisements through social media and planning of events (ideally a week before they occur)
  - 2. Better leveraging and improving HackSoc's digital presence on social media and throughout the student union. As a terminally online GenZ, I unfortunately know my way around most social media.
  - 3. Collaborations with University departments such as Computer Science, Engineering, Sciences, etc. This would also have the benefit of first exposing HackSoc as a society to upcoming cohorts. Currently, we are already collaborating with the Computer Science department in marketing our events and HackNotts through welcome week, the mentor scheme, and the CS weekly newletters - from my view, it already has a massive impact with many more first years involving themselves in HackSoc compared to last year.
  - 4. Weekly newsletter and updates of what the society has done that week along with upcoming events. This would otherwise keep members informed of the events that are going on or they have missed (to entice them :)).
- **Better Outreach and Diversity**; I want to extend the outreach of HackSoc beyond Computer Science. At the moment, HackSoc is more known to be the "Computer Science" society amongst students rather than a general technology society. Quoting the constitution; one of HackSoc's aims is *"To provide opportunities for the Students’ Union “hackers” and technologists from all backgrounds and interests within technology to interact with one another."* with Hackers referring to anyone who like programmable systems and takes joy in solving problems related to them in playful or creative ways. This obviously extend beyond people in Computer Science yet we don't often see that. As the Marketing Secretary, I do hope to resolve this by having collaborations with departments outside of Computer Science and maybe hold events besides programming such as robotics, electronics, and social events (rip compsoc).
- **Creating a more beginner friendly environment for newer and diverser audiences;** Relating to the point above; I often hear that HackSoc is known as the "nerd" society out of the entire student union. Techincally it is true from looking at past events and I wouldn't necessarily think of it as a bad thing, but it also got me thinking whether that would deter newer or potential members from participating in the society. Talking to some people, I see that it was somewhat the case with many stating that *they would want to take part but they think that the workshops are too out of their depth*.  
  - I hope to remedy this through better emphasis in marketing that HackSoc can be a society where everyone and anyone with some interest in digital technology can learn. 
  - In addition to collaborating with university departments, I also want to collaborate with societies and groups such as Women in Computing, BAME, etc. in holding workshops and events.
  - Although it wouldn't be my job, I also want to look into more industry workshops. CompSoc's AWS event had a massive turnout and was extremely popular, along with HackSoc's intro to python. gamedev, and cybersec workshops in 2022. 
- **Welcome Fair:** One thing I want to do with Welcome Fair is to put on some cool displays and demos relating to what the society does. This year's welcome fair booth was rather lackluster compared to other societies' booths and I believe that having something physical to show can go along way in making good first impressions. Nothing too crazy ~~considering the state of our budget~~ but something like the Bad Apple display from when TNMOC was sponsoring, some of the hackathon projects from past hackathons, or some cool software project would go along way in creating a better initial impression of the society. 
- ***Plus anything else that comes to my mind.***

## Why should you vote for me?
1. As mentioned, I have first hand experience as a committee member in helping to run HackSoc. This have given me some exposure to the feedback some members had, and ideas on where HackSoc could improve.
2. I have also helped to organise HackNotts and helped to bring new additions such as HNTV amongst other things.
3. In addition, I've also been working as the Graphic Officer for the current committee and have experience in graphic design. I've taken some feedback from members and committee members and hope to implement them in the coming year regarding graphics if elected. On an unrelated note, I just like to draw (something something graphic design is my passion)
4. I am very proactive within the Computer Science department and HackSoc. Currently, I am a mentor for the 2023/24 cohort and an upcoming senior mentor for the 2024/25 cohort; during then I have help ran workshops and events for the first year students, and mentoring sessions for tutees if they had asked. Outside of the department, I am also a course rep and have been heavily involved in the feedback of modules such as DBI and DMS - this has resulted in said modules being revamped to better fit student feedback (whether the module has been improved or not is not for me to say :) ).
5. I am also very friendly and detail/data oriented person. Which I think can come across well in a marketing job where the individual details matter in attracting a larger audience.

## In Summary
In summary, I want to create better outreach of the society beyond Computer Science and an environment where beginners or learners are more comfortable to participate in instead of being disuaded. All which can be done with the power of marketing :)

uhhhhhhhhhhhh, something something data is the new currency of business, hacksoc is cool, vote for me and you'll get a free bowl of my chicken noodle soup :)

[ok now here's the real manifesto ](https://www.bannedthought.net/China/Individuals/XiJinping/XiJinping-TheGovernanceOfChina.pdf)

**Comment by UnicornsOnLSD** [comment]
> The maid dress is forever encased in my wardrobe and it stares at me in disappointment everytime I open it.

skill issue

**Comment by sanamorii** [comment]
> > The maid dress is forever encased in my wardrobe and it stares at me in disappointment everytime I open it.
> 
> skill issue

You're to blame for this

**Comment by KneeShe-2004** [comment]
where is my course rep credit 🤬😡💢🤬🤬💢

**Comment by UnicornsOnLSD** [comment]
> > > The maid dress is forever encased in my wardrobe and it stares at me in disappointment everytime I open it.
> > 
> > 
> > skill issue
> 
> You're to blame for this

![thisishacksoc](https://github.com/HackSocNotts/nominations/assets/44349936/c237ae84-98ab-4a9f-994a-17c2c1300a0a)
